tests/functional/lib/helpers.js

```diff
@@ -762,7 +762,6 @@ define([
             return click('form input.marketing-email-optin').call(this);
           }
         })
-
```

Comment:
I removed this section because with it we were unable to click on checkboxes. I'm not sure what the best way to go here is.

My suspicion is the click on the "#resend" element is not causing the event handler to always fire since the element has "opacity: 0" to start, and uses CSS animations to fadein after 5 seconds, but I don't know how to test that theory.